hudi-client/src/test/java/org/apache/hudi/TestHoodieClientOnCopyOnWriteStorage.java

```diff
@@ -274,6 +275,15 @@ private void testUpsertsInternal(HoodieWriteConfig hoodieWriteConfig,
     updateBatch(hoodieWriteConfig, client, newCommitTime, prevCommitTime,
         Option.of(Arrays.asList(commitTimeBetweenPrevAndNew)), initCommitTime, numRecords, writeFn, isPrepped, true,
         numRecords, 200, 2);
+
+    // Delete 1
+    prevCommitTime = newCommitTime;
+    newCommitTime = "005";
+    numRecords = 50;
+
+    deleteBatch(hoodieWriteConfig, client, newCommitTime, prevCommitTime,
```

Comment:
I couldn't pinpoint a test-case where we create a brand new hoodie write client and write config (which does not have previous avro schema). If there is one already, we are good.

Reply:
I wrote a test(TestHoodieClientOnCopyOnWriteStorage#testDeletesWithoutInserts), but the config had record schema. Fixed it to have null schema.